src/cropduster.js

```diff
@@ -195,118 +203,149 @@ const CD = {
   },
 
   get: function(url, options, callback) {
-    var args = Array.prototype.slice.call(arguments);
+    const args = Array.prototype.slice.call(arguments);
 
     url = args[0];
-    callback = args.pop();
-    options = args[1] || {};
+    if (typeof options === 'function') {
```

Comment:
I haven't run this code yet, but just looking at it, it looks like we set `options = args[1] || {}`, which sets it to the function. It'll probably _work_ but I think the intention is to set `callback = options; options = {}` since the user didn't pass actual options?